demux.c

```diff
@@ -544,6 +578,13 @@ static void demux_Handle( block_t *p_ts )
     if ( p_pid->info.i_first_packet_ts == 0 )
         p_pid->info.i_first_packet_ts = i_wallclock;
 
+    if ( p_pid->i_sid > 0 )
```

Comment:
Only necessary if printing bitrates is enabled (otherwise there would be unnecessary extra overhead with FindSID).